<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateShapeTopology</class>
 <widget class="QDialog" name="GenerateShapeTopology">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Shape Topology</string>
  </property>
  <widget class="Q3ButtonGroup" name="constrainingLayerButtonGroup">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>1</y>
     <width>578</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLabel" name="constrainingLayerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>42</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Feature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFileLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>43</y>
      <width>325</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inputBrowseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>42</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3ButtonGroup" name="outputButtonGroup">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>108</y>
     <width>578</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="outputFileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Poly File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFileLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>46</y>
      <width>325</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outputBrowseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>45</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>228</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>228</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>228</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3TextBrowser" name="MessageLog" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>280</y>
     <width>570</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inputFileLineEdit</tabstop>
  <tabstop>inputBrowseButton</tabstop>
  <tabstop>outputFileLineEdit</tabstop>
  <tabstop>outputBrowseButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
